Asset/AbstractAssetManager.php

```diff
@@ -220,6 +221,11 @@ protected function actionWhenComposerDependenciesAreAlreadyInstalled($names)
     protected function buildCommand($defaultBin, $action, $command)
     {
         $bin = $this->config->get('manager-bin', $defaultBin);
+        
```

Comment:
Move this section after the `$options = ...` line. Thanks.

Reply:
I have changed that. Is now everything ok?